server/chat-commands/admin.ts

```diff
@@ -275,19 +275,18 @@ export const commands: Chat.ChatCommands = {
 		this.checkCan('addhtml', null, room);
 		if (!target) return this.parse("/help pminfobox");
 
-		target = this.splitTarget(target);
-		this.checkHTML(target);
-		const targetUser = this.targetUser!;
-		this.checkPMHTML(targetUser);
+		const {targetUser, targetUsername, rest} = this.splitUser(target);
+		this.checkHTML(rest);
+		this.checkPMHTML(targetUser, targetUsername);
 
 		// Apply the infobox to the message
 		target = `/raw <div class="infobox">${target}</div>`;
-		const message = `|pm|${user.getIdentity()}|${targetUser.getIdentity()}|${target}`;
+		const message = `|pm|${user.getIdentity()}|${targetUser!.getIdentity()}|${target}`;
```

Comment:
Instead of needing `targetUser!`, we could have a `requireUser` which automatically throws `Chat.ErrorMessage` if the user doesn't exist.

I bet it would save a lot of code.